department-of-veterans-affairs/vets-website
# CT: prod not displaying updated cumulative Service Options

## Description
Based on the text compare between if/else branches changes were made to cleanup the component and resolve the issue of the feature flag `gibctCh33BenefitRateUpdate` not being passed down independently of value of `gibctFilterEnhancement`
![image](https://user-images.githubusercontent.com/1094999/89329947-8d59da00-d65d-11ea-82b2-505ed68ad313.png)

## Testing done
To test have the feature flags below:
- `gibctFilterEnhancement` off
- `gibctCh33BenefitRateUpdate` and `gibct_estimate_your_benefits ` on

Then go to CT Landing Page > Search Results and check "Cumulative Post 9/11" has values as expected in https://github.com/department-of-veterans-affairs/va.gov-team/issues/11472

## Screenshots
![Screen Shot 2020-08-04 at 1 51 03 PM](https://user-images.githubusercontent.com/50601724/89327230-9779d980-d659-11ea-9a21-6ce256cf7c88.png)


## Acceptance criteria
- [x] Correct options are showing

## Definition of done
